tests/attention/test_dllm_vs_flex_attention.py

```
@@ -0,0 +1,1262 @@
+    """
+    Main benchmark: FlashInfer Block Extend (Ragged + Paged) vs Flex Attention
+
+    Scenario: Each request has Q length = qo_len, KV length = total_kv_len
+              q_offset = total_kv_len - qo_len (simulates the last step of incremental prefill)
+    """
+    device = torch.device("cuda:0")
+    sm_scale = 1.0 / math.sqrt(head_dim)
+    q_offset = total_kv_len - qo_len
+
+    print(f"\n{'='*80}")
+    print(f"FlashInfer Block Extend vs PyTorch Flex Attention")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)